packages/synapse-react-client/src/components/SynapseTable/SynapseTable.tsx

```diff
@@ -716,13 +716,6 @@ export class SynapseTable extends React.Component<
           const isFacetSelection: boolean =
             facetIndex !== -1 && facets[facetIndex].facetType === 'enumeration'
           const facet = facets[facetIndex] as FacetColumnResultValues
-          const isSelectedSpanClass = isSelected
-            ? 'SRC-primary-background-color SRC-anchor-light'
-            : ''
-          const isSelectedIconClass = isSelected
-            ? 'SRC-selected-table-icon tool-icon'
-            : 'SRC-primary-text-color tool-icon'
-          const sortSpanBackgoundClass = `SRC-tableHead SRC-hand-cursor SRC-sortPadding SRC-primary-background-color-hover  ${isSelectedSpanClass}`
```

Comment:
I found that most of these are not really necessary and can be kept to some simple MUI styling